bin/cert-generator-wrapper.sh.in

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+
+usage() {
+	cat <<EOF
+$(basename "${0}"): SCF Certificate Generator
+
+  -d <domain>:     Sets the top level domain for the cluster
```

Comment:
Per line 42 below this is required information. IMHO the help should mention that. Or make it a non-option argument.